src/Evolution/Systems/ScalarAdvection/System.hpp

```diff
@@ -0,0 +1,20 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+/*!
+ * \ingroup EvolutionSystemsGroup
+ * \brief Items related to evolving the scalar advection equation.
+ *
+ * \f{align*}
+ * \partial_t U + \nabla \cdot (v U) = 0
+ * \f}
+ *
+ * The velocity field \f$v\f$ is hard-coded in the current implementation since
+ * ScalarAdvection system is only used for testing limiters.
+ */
+namespace ScalarAdvection {
+template <size_t Dim>
+struct System {};
```

Comment:
Question for @nilsdeppe: Should we add docstrings to the system as well, so it shows up in the documentation?

Perhaps it would be appropriate to move the `\ingroup EvolutionSystemsGroup` and most of the technical documentation from the namespace to the system (like the info on the velocity field above, dox for template parameters if the system has any, etc), and have the namespace only describe the physics of the scalar advection equation. What do you think?

Reply:
We have not done that for other systems. I think it actually be good to move _all_ the documentation of the functions to the namespace. Currently many systems have some dox on the namespace, some on `Fluxes` structs, etc. and this makes it difficult to find the actual evolution equations. Having said that, add a dox comment to make sure it shows up seems like a good idea, but I don't think it should be more than `/// \brief The system class for the Scalar Advection evolution system`

I don't think that's a good idea. I think it's better to make the entire namespace be found easily in the documentation so that all the parts of the system are readily available. If only the `System` class is in that group, then it'll be hard to find all the other parts of the system.  Aside from the dox being spread out over multiple structs, I actually quite like the current way of documenting these.

Reply:
Ok, documenting all system-specific things in the namespace also sounds good to me 👍